samples/cellular/fmfu_smp_svr/README.rst

```diff
@@ -71,14 +81,14 @@ After programming the sample to your development kit, test it by performing the
      .. parsed-literal::
         :class: highlight
 
-        python update_modem.py mfw_nrf9160_1.2.2.zip /dev/ttyACM0 *115200*
+        python update_modem.py mfw_nrf9160_1.3.5.zip /dev/ttyACM0 *115200*
```

Comment:
This will not work with the nRF9161DK. It needs MFW 2.x.

Reply:
This was an example for the nRF9160 DK. May be we could add similar one for the nRF9161 DK (using tabs) with MFW 2.x?

Reply:
Yes, mfw 2.x was not released yet when I made the PR, but there is a beta now on the nRF9161 product page, so we can add it now!